basic-commits/README.md

```diff
@@ -1,11 +1,13 @@
 # Git Kata: Basic Commits
-This kata will introduce you to the commands `git add` and `git commit`.
+This kata will introduce you to the commands `git add` and `git commit`. 
+
+This is a very introductory kata. if you have used `git status`, `git log --oneline --graph`, `git add` and `git commit` extensively you should probably skip it.
 
 You can look at the bottom of this file, if you have not yet done basic git configuration.
 
 ## Setup:
 
-1. Run `. setup.sh` (or `.\setup.ps1` in PowerShell)
+1. Run `. setup.sh` (or `.\setup.ps1` in PowerShell) and `cd exercise`
```

Comment:
The whole point of sourcing the script (i.e. running the script with `dot space scriptname`, or `source scriptname`) is that the script is run in the current shel process, and thus leaves you inside the exercise folder when done. So if you run the shel script the right way, it makes no sense and is not possible to `cd exercise` afterwards.'

The rest of the change looks good to me.

Reply:
In powershell it might be different - I have no idea.

Reply:
Removed the `cd`.
(I thought the `. setup.sh` was a typo and you meant `.\setup.sh`. Hence my confusion)

This is a separate comment, but the readmes were recently changed to use numbered lists with increasing numbers.

 1. Use `git status` to see which branch you are on.
 2. What does `git log` look like?

Instead of

 1. Use `git status` to see which branch you are on.
 1. What does `git log` look like?

I plan to submit changes where I add items to the task and its going to look like I made a lot of changes when I only added a line or two.